### PR TITLE
FYLE-3tfhyt: Split expense action sheet

### PR DESCRIPTION
**Pull request by Dimple16**
Screenshot 2021-06-17 at 4 13 29 PM

### CODE REVIEW

File: src/app/fyle/add-edit-expense/add-edit-expense.page.ts [hunk 1]
```diff
@@ -554,21 +555,38 @@ export class AddEditExpensePage implements OnInit {
...
+        });
+      }
 
-      if (data && data.type) {
-        this.openSplitExpenseModal(data.type);
+      if (areCostCentersAvailable) {
+        this.actionSheetButtons.push({
+          text: 'Cost center',
+          handler: () => {
+            this.openSplitExpenseModal('cost centers')
```
Comment (iamtarun1993) [inline src/app/fyle/add-edit-expense/add-edit-expense.page.ts:578]: please verify once this is 'cost centers' not 'cost_centers'

File: src/app/fyle/add-edit-expense/add-edit-expense.page.ts [hunk 2]
```diff
@@ -554,21 +555,38 @@ export class AddEditExpensePage implements OnInit {
...
+            this.openSplitExpenseModal('projects')
+          }
+        });
+      }
 
-      if (data && data.type) {
-        this.openSplitExpenseModal(data.type);
+      if (areCostCentersAvailable) {
+        this.actionSheetButtons.push({
+          text: 'Cost center',
```
Comment (iamtarun1993) [inline src/app/fyle/add-edit-expense/add-edit-expense.page.ts:576]: 'Cost Center',

**Review by iamtarun1993 — APPROVED** [review]

File: src/app/fyle/add-edit-expense/add-edit-expense.page.ts (hunk 1 above)
Comment (Dimple16) [reply src/app/fyle/add-edit-expense/add-edit-expense.page.ts:578]: confirmed - `cost centers`

File: src/app/fyle/add-edit-expense/add-edit-expense.page.ts (hunk 2 above)
Comment (Dimple16) [reply src/app/fyle/add-edit-expense/add-edit-expense.page.ts:576]: added same as design

**Pull request merged**